app/otherOperations.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function (passport) {
+    return setupFunctions(passport);
+};
+
+var setupFunctions = function (passport) {
+    var exports = {};
+
+    exports.getCategories = function (req, res) {
+        var output = [];
+        for (var i = 0; i < global.questionCategories.length; i++) {
```

Comment:
If we decide to use lodash, we can replace this with ``_.foreach(global.questionCategories)``